.github/workflows/ci-stable-release.yml

```
@@ -51,16 +51,22 @@ jobs:
         id: update
         run: |
           STABLE_VERSION=`cat STABLE-RELEASE`
-          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/Chart.yaml
-          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/Chart.yaml
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/KubeArmor/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/KubeArmor/Chart.yaml
+
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/KubeArmorOperator/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/KubeArmorOperator/Chart.yaml
+
           echo "STABLE_VERSION=$STABLE_VERSION" >> $GITHUB_OUTPUT
 
       - name: Create PR to update Helm chart version in KubeArmor repo
         uses: peter-evans/create-pull-request@v5
         with:
           branch: update-helm-${{ steps.update.outputs.STABLE_VERSION }}
-          add-paths: "deployments/helm/Chart.yaml"
+          add-paths: "deployments/helm/KubeArmor/Chart.yaml, deployments/helm/KubeArmorOperator/Chart.yaml"
```

```suggestion
          add-paths: "deployments/helm/*/Chart.yaml, deployments/helm/README.yaml"
``` 

This wouldn't do? If not let's leave it.

---

This one actually creates a PR in kubearmor repo and updates chart versions after a stable release has been made. It doesn't do any changes in the charts repo.

---

ahh my bad

---

This is where we update charts repo - https://github.com/kubearmor/KubeArmor/blob/main/.github/workflows/ci-latest-helm-chart-release.yaml#L29